README.md

```diff
@@ -1,61 +1,53 @@
 # ansible-aci
 
-The ansible-aci project provides an Ansible collection for managing and automating your Cisco ACI environment. It consists of a set of modules and roles for performing tasks related to ACI.
+## Description
 
-This collection has been tested and supports ACI 3.2+.
-Modules supporting new features introduced in ACI API in specific ACI versions might not be supported in earlier ACI releases.
+The ansible-aci project provides an Ansible collection for managing and automating your Cisco ACI environment. It consists of a set of modules and roles for performing tasks related to ACI.
 
-*Note: This collection is not compatible with versions of Ansible before v2.8.*
+See the [Cisco.Aci collection index](https://galaxy.ansible.com/ui/repo/published/cisco/aci/content/) for a full list of modules and plugins.
 
 ## Requirements
 
-Ansible v2.14 or newer
+- Ansible v2.15 or newer
+- Python v3.10 or newer
 
-## Install
+*Note: This collection is not compatible with versions of Ansible before v2.8.*
 
-Ansible must be installed
+Follow the [Installing Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) guide for detailed instructions.
 
-```sh
-sudo pip install ansible
-```
+## Installation
 
-Install the collection
+Before using this collection, you need to install it with the Ansible Galaxy command-line tool:
 
 ```sh
 ansible-galaxy collection install cisco.aci
 ```
 
-## Use
+You can also include it in a requirements.yml file and install it with ansible-galaxy collection install -r requirements.yml, using the format:
 
-Once the collection is installed, you can use it in a playbook by specifying the full namespace path to the module, plugin and/or role.
+```yaml
+collections:
+  - name: cisco.aci
+```
 
-```yml
-- hosts: aci
-  gather_facts: no
+Note that if you install any collections from Ansible Galaxy, they will not be upgraded automatically when you upgrade the Ansible package.
+To upgrade the collection to the latest available version, run the following command:
 
-  tasks:
-  - name: Add a new EPG
-    cisco.aci.aci_epg:
-      hostname: apic
-      username: admin
-      password: SomeSecretPassword
-      tenant: production
-      ap: intranet
-      epg: web_epg
-      description: Web Intranet EPG
-      bd: prod_bd
-    delegate_to: localhost
+```sh
+ansible-galaxy collection install cisco.aci --upgrade
 ```
 
-## Optimizing Playbooks
+You can also install a specific version of the collection, for example, if you need to downgrade when something is broken in the latest version (please report an issue in this repository). Use the following syntax to install version 1.0.0:
```

Comment:
The `for example, if you need to downgrade when something is broken in the latest version` does not sound great. I would rephrase that.

Reply:
This is taken right from the RedHat example but I'll change it. We don't need to follow it exactly.

Reply:
Re-worded. I took out the specific example of why someone would potentially attempt to downgrade.